### ISSUE TITLE
`REFERENCES` constraints on table creation leads to duplicate creation of `pgmemento_audit_id` column

### ISSUE THREAD

**Issue by phryneas** [pgMemento/pgMemento#70]
Hey there :)

I believe I have found a bug where a trigger triggers once too often or something like that.

This is something I noticed when updating the tests for `postgraphile-audit-plugin` to 0.7 and it also occurs on the current master.

It can be repoduced by running `INSTALL_PGMEMENTO.sql` followed by:

```sql

CREATE SCHEMA postgraphile_audit_plugin;

SELECT pgmemento.init(
    'postgraphile_audit_plugin', -- schema_name
    'pgmemento_audit_id', -- audit_id_column_name
    TRUE, -- log_old_data
    TRUE, -- log_new_data
    FALSE, -- log_state
    TRUE, -- trigger_create_table
    '{}' -- except_tables
  );

CREATE TABLE postgraphile_audit_plugin.users (
	id SERIAL PRIMARY KEY
);

CREATE TABLE postgraphile_audit_plugin.users_emails (
	user_id INT NOT NULL REFERENCES postgraphile_audit_plugin.users(id)
);
```

The second `CREATE TABLE` statement will fail with
```
psql:__tests__/setupSchema.sql:27: ERROR:  column "pgmemento_audit_id" of relation "users_emails" already exists
CONTEXT:  SQL statement "ALTER TABLE postgraphile_audit_plugin.users_emails ADD COLUMN pgmemento_audit_id BIGINT DEFAULT nextval('pgmemento.audit_id_seq'::regclass) UNIQUE NOT NULL"
PL/pgSQL function create_table_audit_id(text,text,text) line 8 at EXECUTE
SQL statement "SELECT pgmemento.create_table_audit_id($1, $2, $3)"
PL/pgSQL function create_table_audit(text,text,text,boolean,boolean,boolean) line 41 at PERFORM
SQL statement "SELECT pgmemento.create_table_audit(
          tablename,
          schemaname,
          current_default_column,
          current_log_old_data,
          current_log_new_data,
          FALSE
        )"
PL/pgSQL function table_create_post_trigger() line 42 at PERFORM
```

To reproduce this, you can use our test setup:

Just clone https://github.com/mayflower/postgraphile-audit-plugin and run `yarn run-tests` (you'll need `yarn` and `docker` installed).

**Comment by FxKu** [comment]
Looks pretty much the same like #68 which I fixed last week. Please, try with the master. I hope to find time to fix the open issue to draft a 7.1 release soonish.

**Comment by phryneas** [comment]
I could have *sworn* that I had also tested this with the current master, but that fixes it. Sorry for the noise!
